Breaking up envvars and config settings into separate settings pages (#1028)
minio/docs: source/reference/minio-mc-admin/mc-admin-config.rst

   Roll back changes to configuration keys to a previous point in history.

   Does not affect configurations defined by environment variables.

Configuration Settings
----------------------

> For the list of possible settings, descriptions, and options to use when configuring them, see :ref:`minio-server-configuration-settings`.

Reviewer: Maybe?
```suggestion
For a list of available configuration settings, see :ref:`minio-server-configuration-settings`.
```
